src/icons/ConfirmIcon.tsx

```diff
@@ -0,0 +1,18 @@
+const ConfirmIcon = () => {
+    return (
+        <svg
```

Comment:
If you'd like to save some space you could do 2 things.

1. `() => { return X }` → `() => X`, or
2. `export default function ConfirmIcon() { return X }`